### ISSUE TITLE
ModelView download method crashes

### ISSUE THREAD

**Issue by befuhro** [dpgaspar/Flask-AppBuilder#1964]
### Environment

Flask-Appbuilder version: 4.1.6

pip freeze output:
alembic==1.8.1
apispec==3.3.2
attrs==22.1.0
Babel==2.11.0
click==8.1.3
colorama==0.4.6
contourpy==1.0.6
cycler==0.11.0
dnspython==2.2.1
email-validator==1.3.0
Flask==2.2.2
Flask-AppBuilder==4.1.6
Flask-Babel==2.0.0
Flask-JWT-Extended==4.4.4
Flask-Login==0.6.2
Flask-Migrate==4.0.0
Flask-SQLAlchemy==2.5.1
Flask-WTF==1.0.1
fontawesomefree==6.2.1
fonttools==4.38.0
greenlet==2.0.1
idna==3.4
itsdangerous==2.1.2
Jinja2==3.1.2
jsonschema==4.17.3
kiwisolver==1.4.4
Mako==1.2.4
MarkupSafe==2.1.1
marshmallow==3.19.0
marshmallow-enum==1.5.1
marshmallow-sqlalchemy==0.26.1
matplotlib==3.6.2
numpy==1.23.5
packaging==21.3
pandas==1.5.2
Pillow==9.3.0
prison==0.2.1
pyasn1==0.4.8
pyasn1-modules==0.2.8
PyJWT==2.6.0
pyparsing==3.0.9
pyrsistent==0.19.2
python-dateutil==2.8.2
python-ldap==3.4.3
pytz==2022.6
pywaffle==1.1.0
PyYAML==6.0
six==1.16.0
SQLAlchemy==1.4.44
SQLAlchemy-Utils==0.38.3
Werkzeug==2.2.2
WTForms==3.0.1

### Describe the expected results
Downloading a file via ModelView

### Describe the actual results
Internal Server Error

```pytb
Traceback (most recent call last):
  File "/root/CSIRT/ucx/.venv/lib/python3.10/site-packages/flask/app.py", line 2548, in __call__
    return self.wsgi_app(environ, start_response)
  File "/root/CSIRT/ucx/.venv/lib/python3.10/site-packages/flask/app.py", line 2528, in wsgi_app
    response = self.handle_exception(e)
  File "/root/CSIRT/ucx/.venv/lib/python3.10/site-packages/flask/app.py", line 2525, in wsgi_app
    response = self.full_dispatch_request()
  File "/root/CSIRT/ucx/.venv/lib/python3.10/site-packages/flask/app.py", line 1822, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/root/CSIRT/ucx/.venv/lib/python3.10/site-packages/flask/app.py", line 1820, in full_dispatch_request
    rv = self.dispatch_request()
  File "/root/CSIRT/ucx/.venv/lib/python3.10/site-packages/flask/app.py", line 1796, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
  File "/root/CSIRT/ucx/.venv/lib/python3.10/site-packages/flask_appbuilder/security/decorators.py", line 133, in wraps
    return f(self, *args, **kwargs)
  File "/root/CSIRT/ucx/.venv/lib/python3.10/site-packages/flask_appbuilder/views.py", line 641, in download
    return send_file(
TypeError: send_file() got an unexpected keyword argument 'attachment_filename'
```

### Steps to reproduce
Implementing a model that has a download feature

### Solution
Replacing the attachment_filename parameter with download_name
As shown in the change here https://flask.palletsprojects.com/en/2.2.x/changes/#version-2-2-0

**Comment by dpgaspar** [comment]
Hi,

Thank you for the report, a fix is already merged on master but not yet released. Meanwhile if possible you can test/use current master.

**Comment by dpgaspar** [comment]
fix was released on 4.2.0
